Model/Paypal.php

```diff
@@ -361,12 +361,46 @@ protected function getReferenceIdForRefund($payment)
      */
     public function isAvailable(\Magento\Quote\Api\Data\CartInterface $quote = null)
     {
+        $this->_logger->debug('---------------- isAvailable - START OF REQUEST----------------');
```

Comment:
I have the same concern with these debug statements, I don't know if it makes sense to leave these debug statements in the codebase. But please let me know if I'm missing something beneficial from leaving these debug statements in

Reply:
These are paired with the 'test' functions to output valuable variables to the debug console for debugging.

Your choice on to leave them, comment them or remove them entirely.